common/models/middleware.js

```diff
@@ -16,6 +16,23 @@ module.exports = function(Middleware) {
     }
     return phase;
   };
+  Middleware.getPhaseFromId = function(id) {
+    const parts = id.split('.');
+    if (parts.length && parts.length > 1) {
```

Comment:
parts will always have length here since its an array. If it doesn't it will have thrown an error in the split